.chloggen/jaeger_ui_routes.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack, tempomonolithic
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Register missing Jaeger UI routes
```

Comment:
Where are you implementing this? I only see some changes in manifest files.

Reply:
That's implemented in https://github.com/observatorium/api/pull/781
I thought it makes sense to copy the actual reason/fix to the operator changelog, versus just writing "bump gateway" in the operator changelog.